lib/polish_number.rb

```diff
@@ -21,86 +21,181 @@ module PolishNumber
 
   MILLIONS = {:one => 'milion', :few => 'miliony', :many => 'milionów'}
 
+  BILLIONS = {:one => 'miliard', :few => 'miliardy', :many => 'miliardów'}
+
+  CENTS = [:auto, :no, :words, :digits]
+
   CURRENCIES = {
```

Comment:
Again `UNITS` would make much more sense, if you want to have support for arbitrary words, like in the next line.